src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -54,7 +54,8 @@
         {
           "condition": "(ExcludeLaunchSettings)",
           "exclude": [
-            "Properties/launchSettings.json"
+            "Client/Properties/launchSettings.json",
+            "Server/Properties/launchSettings.json"
```

Comment:
I believe you may have added a baseline test for this which may need to be updated?

Reply:
I added [ this](https://github.com/dotnet/aspnetcore/blob/08760b3936e2ac6fcd413e598da7af258a33ddc6/src/ProjectTemplates/test/Templates.Tests/template-baselines.json#L1814) baseline test for `blazorwasm-empty`, in which `launchSettings.json` is already excluded.

Reply:
Ah okay I see. Can we please add another test in that case for `new blazorwasm-empty --exclude-launch-settings --hosted`. Reason being, the directory structure of the solution (and thus the `launchSettings.json`) is different and want to make sure we aren't running into some edge case with it.

Reply:
Sounds good. I just added the test for this.